packages/lab/src/overlay/OverlayPanel.tsx

```diff
@@ -43,30 +46,25 @@ export const OverlayPanel = forwardRef<HTMLDivElement, OverlayPanelProps>(
       });
     };
 
-    if (!openState) return <></>;
-
     return (
-      <FloatingOverlay>
-        <FloatingComponent
-          open={openState}
-          className={clsx(withBaseName(), className)}
-          aria-modal="true"
-          aria-labelledby={`${id}-header`}
-          aria-describedby={`${id}-content`}
-          {...getOverlayProps()}
-          ref={handleRef}
-          width={width}
-          height={height}
-          top={top}
-          left={left}
-          position={position}
-          focusManagerProps={{
-            context: context,
-          }}
-        >
-          <OverlayPanelBase {...rest} />
-        </FloatingComponent>
-      </FloatingOverlay>
+      <FloatingComponent
+        open={openState}
+        className={clsx(withBaseName(), className)}
+        aria-modal="true"
+        {...getOverlayProps()}
+        ref={handleRef}
+        width={width}
+        height={height}
+        top={top}
+        left={left}
+        position={position}
+        focusManagerProps={{
+          context: context,
+        }}
+        {...a11yProps}
```

Comment:
For the title to be announced by the screen reader the aria-described by needs to be attach to the FloatingComponent